src/lib/debounce.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function debounce(fn, delay) {
```

Comment:
`yarn add lodash.debounce`?

Reply:
ah I didn't know that you couldnt install the debounce function separately from the entire Lodash library